src/org/usfirst/frc/team2706/robot/commands/autonomous/Priority.java

```diff
@@ -92,43 +92,25 @@ public Priority(Command linkedCommand) {
      * @return
      */
     public boolean getPossible() {
-        if (guaranteedPriority || DriverStation.getInstance().getGameSpecificMessage().equals("")) {
+        if (guaranteedPriority
+                        || DriverStation.getInstance().getGameSpecificMessage().length() >= 2) {
```

Comment:
This should be less than, but I'll leave it until there's something else to change